assets/src/settings-page/welcome.js

```diff
@@ -92,7 +92,7 @@ export function Welcome() {
 					{
 						customizerLink && (
 							<a className="components-button is-link" href={ customizerLink } rel="noreferrer">
-								{ __( 'Visit Customizer', 'amp' ) }
+								{ __( 'Customize Theme', 'amp' ) }
```

Comment:
I'm wondering if this could be confusing for the user. I think specifying what theme they're customizing would prevent any potential confusion.

```suggestion
								{ __( 'Customize Reader Theme', 'amp' ) }
```